app/src/assets/localization/en/pipette_wizard_flows.json

```
@@ -19,7 +19,7 @@
   "detach": "Detaching Pipette",
   "calibration_probe_touching": "The calibration probe will touch the sides of the calibration divot in slot {{slotNumber}} to determine its exact position",
   "choose_pipette": "Choose a pipette to attach",
-  "attach_this_pipette": "Attach this pipette",
+  "next": "next",
```

```suggestion
  "next": "Next",
```
I think this is button text, so it should be sentence case.

---

thanks for noticing this, it will be capitalized in the component! I think the best practice is to not capitalize anything in the i18n strings and have the component capitalize it but we do a pretty bad job of sticking with that.